### DISCUSSION TITLE
It does not work properly after updating 24H2

### DISCUSSION THREAD

**Discussion by Minanl** [rocksdanister/lively#2412] [Q&A]
After updating 24H2, the software can start normally after boot.
However, it only works properly on 1 screen, and the other screen displays incorrectly.
Left and right side and upper side, 3 side missing does not show.
After repeated testing, when the wallpaper is set up on one screen, the wallpaper works properly, and then set the wallpaper on the other screen, the wallpaper on the previous screen will appear as described above.

![IMG_2141](https://github.com/user-attachments/assets/50c3862b-81f8-45f3-87b1-553ec5059c7d)

**Comment by rocksdanister** [comment]
24H2 is still in beta/preview channel right?

This issue should be patched in the Microsoft Store version, for installer users can patch themselves:
https://github.com/rocksdanister/lively/issues/2074#issuecomment-2146541025

**Reply by Minanl** [reply]
Yes, problem solved. Thank you
